modules/configuring-an-authentication-provider-in-central.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * cli/getting-started-cli.adoc
+:_content-type: PROCEDURE
+[id="configuring-an-authentication-provider-in-central_{context}"]
+= Configuring an authentication provider in Central
+
+[role="_abstract"]
+You can configure an authentication provider in Central and initiate the login process with the `roxctl` CLI.
+Set the variable `ROX _ENDPOINT`, initiate the login process with the `roxctl central login` command, select the authentication provider in a browser window, and retrieve the token information from the `roxctl` CLI.
+
+.Procedure
+
+. Select an authentication provider of your choice, such as OpenID Connect (OIDC) with fragment or query mode, and configure it in Central.
```

Comment:
This step confuses me. I think that it suggests picking a provider, which you will configure in the following steps. If so, should it be a prerequisite? It's not really a step in how to configure a provider with Central.

```
Prerequisite
* You selected an authentication provider of your choice, such as OpenID Connect (OIDC) with fragment or query mode.
```

Reply:
@dhaus67 WDYT?